pkg/store/cache/caching_bucket.go

```diff
@@ -95,13 +96,17 @@ func NewCachingBucket(b objstore.Bucket, c cache.Cache, chunks CachingBucketConf
 			Name: "thanos_store_bucket_cache_refetched_chunk_bytes_total",
 			Help: "Total number of chunk bytes re-fetched from storage, despite being in cache already.",
 		}, []string{"origin"}),
-		objectSizeRequests: promauto.With(reg).NewCounter(prometheus.CounterOpts{
-			Name: "thanos_store_bucket_cache_objectsize_requests_total",
-			Help: "Number of object size requests for objects.",
+		objectAttrsRequests: promauto.With(reg).NewCounter(prometheus.CounterOpts{
+			Name: "thanos_store_bucket_cache_object_attrs_requests_total",
```

Comment:
Can we check if kube-thanos used the old metric name?

Reply:
I've rebased from `master`. Compared to `master` there're no more changes to metrics now.